Readme.md

```diff
@@ -47,3 +47,19 @@ Where:
 - `token` - the token to access the Qase API
 - `testrun-id` - the id of the test run in Qase
 - `output` - the file to save the expected results
+
+## How to use github action
+
+To use this tool in the github action, you need to create a workflow file in the `.github/workflows` directory and add
+the following step. For example:
+
+```yaml
+- name: Run Reporter Validator
+  uses: qase-tms/reporters-validator@v1
+  id: run_validator
+  with:
+    project-code: ${{ secrets.QASE_PROJECT_CODE }}
+    token: ${{ secrets.QASE_TOKEN }}
+    testrun-id: 1
```

Comment:
Is this the sample (ideal) run or a new run that we're testing? For the second, we should use a variable, like `${ env.QASE_RUN_ID}`.

Besides that, let's just call this variable `run-id`, because it matches the env variable name `QASE_RUN_ID`, which is used by reporters.